README.md

```diff
@@ -61,10 +62,38 @@ Airflow 1.10.4
 pip install -r requirements.txt
 ```
 
-## Running the tests
+### Running the tests
 ```
 python -m pytest tests/test_ExtractCCLinks.py
 ```
+## Provider API puller module development
+There are a number of scripts in the directory `src/providers/api` which pull image data from various restful apis, and format it into files which are eventually loaded into a database to be indexed for searching on CC Search.  These run in a different environment than the PySpark portion of the project, and so have their own dependency requirements.
+
+### Setup the easy way
+There is a Dockerfile provided in the `src/providers/api` directory.  With docker installed, navigate to that directory and run
+```
+docker build -t cccdev -f Dockerfile.localdev .
```

Comment:
It looks like `Dockerfile.localdev` was missing from this PR, I'm not able to run this command